build/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@
 if(WIN32)
   ROOT_EXECUTABLE(bindexplib
     win/bindexplib/bindexplib.cxx
+    ${ROOT_RC_SCRIPT}
```

Comment:
Instead of adding it to each invocation, could we instead add it to `ROOT_EXECUTABLE` itself?

Reply:
I'll check

Reply:
@Axel-Naumann OK, it works, but do we want all executables (e.g. all tests) having the ROOT icon?